### PR TITLE
1.16.3 Patch

### PR DESCRIPTION
**Pull request by Siuolthepic**
Hoo boy! Has this been a ride. Hopefully I dont have to do this again on Monday, but you never know.
The commit log may be confusing, so let me explain this.
Essentially, the main changes include.
1. Mixin Internal Server got updated a lil, mostly just changing a mapping import and removing a few lines that arnt needed
2. I updated the gradle.properties and build.gradle files.
3. I added a small description. Which I recommend be expanded on after this.
Hopefully you decide to accept this.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -10,8 +10,8 @@ targetCompatibility = JavaVersion.VERSION_1_8
 
 if (isNullOrEmpty(System.getenv("TRAVIS_TAG")) && !isNullOrEmpty(System.getenv('TRAVIS_BUILD_NUMBER'))) {
     version = project.modVersion + '+build.' + System.getenv('TRAVIS_BUILD_NUMBER')
-} else {
-    version = project.modVersion + "+${project.minecraftVersion}"
+} else {	
+    version = project.modVersion + "+${project.minecraftVersion}"
```
Comment (TheGlitch76) [inline build.gradle:14]:
```suggestion
} else {
    version = project.modVersion + "+${project.minecraftVersion}"
```

File: build.gradle [hunk 2]
```diff
@@ -44,6 +44,7 @@ dependencies {
 
     compile "com.dosse.upnp:WaifUPnP:1.1" // the version is likely to never change
     shadowed "com.dosse.upnp:WaifUPnP:1.1"
+    include "com.dosse.upnp:WaifUPnP:1.1"
```
Comment (TheGlitch76) [inline build.gradle:47]: why include this if it's already shadowed?

File: gradle.properties [hunk 3]
```diff
@@ -2,7 +2,7 @@
 org.gradle.jvmargs=-Xmx2G
 
 # Project Properties
-modVersion=0.3.0-SNAPSHOT
+modVersion=0.3.2
```
Comment (TheGlitch76) [inline gradle.properties:5]:
```suggestion
modVersion=0.4.0
```

File: build.gradle (hunk 2 above)
Comment (Siuolthepic) [reply build.gradle:47]: I wasnt able to boot it on production without including it. Ill do some more testing on that regard.

Comment (Siuolthepic) [reply build.gradle:47]: Yeah, if I leave it solely shadowed, the Jar refuses to build it with WaifuPNP in it.

**Review by TheGlitch76 — APPROVED** [review]

**Pull request merged**